protocol/bc/outputid.go

```diff
@@ -0,0 +1,44 @@
+package bc
+
+import (
+	"io"
+
+	"chain/crypto/sha3pool"
+	"chain/encoding/blockchain"
+)
+
+// OutputID identifies previous transaction output in transaction inputs.
+type OutputID struct{ Hash }
```

Comment:
It was originally `type OutputID Hash`, but I suggested embedding the other type since we want to keep all the methods. (We might consider doing this for the other types as well, at some point.)